src/lucky/asset_helpers.cr

```diff
@@ -14,7 +14,7 @@ module Lucky::AssetHelpers
 
     {% if path.is_a?(StringLiteral) %}
       {% if Lucky::AssetHelpers::ASSET_MANIFEST[path] %}
-        {{ Lucky::AssetHelpers::ASSET_MANIFEST[path] }}
+        Lucky::Server.settings.asset_host + {{ Lucky::AssetHelpers::ASSET_MANIFEST[path] }}
```

Comment:
Can you look at a spec for this? Maybe in https://github.com/luckyframework/lucky/blob/500e4bc28786722700672c398b768b3937b204cd/spec/lucky/asset_helpers_spec.cr? Habitat has a `temp_config` method so you can set an asset_host to some production URL, and then test that the asset path has the whole thing.